js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
Seems likely, although I `rm -rf ./_site && npm start` and it still seems to run to that point and then stick.